rust/tw_evm/src/abi/non_empty_array.rs

```diff
@@ -56,6 +56,27 @@ impl<T> NonEmptyArray<T> {
     pub fn into_vec(self) -> Vec<T> {
         self.0
     }
+
+    pub fn get(&self) -> &Vec<T> {
+        &self.0
+    }
+
+    pub fn set(&mut self, value: Vec<T>) {
+        if value.is_empty() {
+            panic!("`NonEmptyArray` must have at least one element");
+        }
+        self.0 = value;
+    }
```

Comment:
Where do you use this method? Also it's critical to avoid panics, consider returning an error instead